MathFunction.java

```diff
@@ -0,0 +1,21 @@
+package com.company.hw01;
+
+public class MathFunction {
+
+    public double getPow(float x, double pow) {
+        double inner_pow = pow;
+        double result = x;
+        if (pow < 0) {
+            inner_pow = -pow;
+        }
+        for (int i = 1; i < inner_pow; i++) {

```

Comment:
you can write here just for (int i = 1; i < Math.abs(pow); i++) {

and you will not need inner_pow  at all

Reply:
> you can write here just for (int i = 1; i < Math.abs(pow); i++) {
> 
> and you will not need inner_pow at all

Oh, it's so obvious, it's weird that I didn't figure it out myself.